openreview/conference/matching.py

```diff
@@ -415,28 +415,38 @@ def _build_config_invitation(self, scores_specification):
                     'conflicts_invitation': {
                         'value': self.conference.get_conflict_score_id(self.match_group.id),
                         'required': True,
-                        'description': 'Invitation to store aggregated scores',
+                        'description': 'Invitation to store conflict scores',
                         'order': 10
                     },
-                    'custom_load_invitation': {
-                        'value': self._get_edge_invitation_id('Custom_Load'),
-                        'required': True,
-                        'description': 'Invitation to store aggregated scores',
-                        'order': 11
-                    },
                     'assignment_invitation': {
                         'value': self.conference.get_paper_assignment_id(self.match_group.id),
                         'required': True,
                         'description': 'Invitation to store paper user assignments',
-                        'order': 12
+                        'order': 11
+                    },
+                    'custom_user_demand_invitation': {
+                        'description': 'Invitation to store custom max number of users required by papers',
```

Comment:
Can we remove max from this description too?

Reply:
Done